docs/conf.py

```diff
@@ -384,6 +384,23 @@ def linkcode_resolve(domain, info):
 nbsphinx_execute = "never"
 # -- Options for LaTeX output ---------------------------------------------
 
+# -- Detect branch name -----------------------------------
+# Works on GitHub Actions, ReadTheDocs, and local fallback
+BRANCH = (
+    os.environ.get("GITHUB_REF_NAME")  # GitHub Actions
```

Comment:
I'm not sure that this is reliable actually - since a lot of contributions come from forks.

Also the link it generates atm is https://github.com/OceanParcels/Parcels/blob/2293/docs/examples/tutorial_unitconverters.ipynb (using the PR number instead). Perhaps we can just remove this?

Reply:
I got your point, @VeckoTheGecko. I'm still trying to figure out how this works on Github (locally is fine, targeting the documentation to the main version).

Reply:
Let's chat f2f in the break :)